xpdacq/utils.py

```diff
@@ -277,7 +277,7 @@ def load(self, saf_num):
 
         self.pd_dict = pd.read_excel(os.path.join(self.src_dir,
                                                   xl_f.pop()),
-                                     skiprows=0)
+                                     skiprows=[1])
```

Comment:
I will assume it is correct, but please check.  Does this skip row 1 or all rows up to 1?  I would rather expect to see `skiprows=[:1]`